openfisca_web_api_preview/openAPI.yml

```diff
@@ -0,0 +1,253 @@
+swagger: "2.0"
+info:
+  title: "{CONTRY_PACKAGE_NAME} Web API"
+  description: "The OpenFisca Web API lets you get up-to-date information and formulas pertaining to the {CONTRY_PACKAGE_NAME} legislation e.g. minimum wage, state scholasrship, social benefits, income taxes…"
+  version: "0.1.0"
+  termsOfService: "https://doc.openfisca.fr/publishing.html"
+  contact:
+    email: "contact@openfisca.fr"
+  license:
+    name: "AGPL"
+    url: "https://www.gnu.org/licenses/agpl-3.0.html"
+host: null
+schemes:
+  - "https"
+tags:
+- name: "Parameters"
+  description: "A parameter is a numeric property of the legislation which can evolve over time."
```

Comment:
Shouldn't “which” be “that”? I read “which” as referring to the legislation, not to the parameter.